kingpin/actors/group.py

```diff
@@ -50,6 +51,7 @@ class BaseGroupActor(base.BaseActor):
 
     all_options = {
         'contexts': (list, [], "List of contextual hashes."),
+        'context-file': (str, '', "A JSON file with a list of contexts."),
```

Comment:
I don't think I like that. I'll try to formulate exactly why, but in general I'm not a fan of overloading a variable.